apps/api/src/app/environments/usecases/update-environment/update-environment.usecase.ts

```diff
@@ -32,4 +41,24 @@ export class UpdateEnvironment {
       { $set: updatePayload }
     );
   }
+  async shouldUpdateEchoConfiguration(command: UpdateEnvironmentCommand): Promise<boolean> {
+    if (process.env.NOVU_ENTERPRISE === 'true' || process.env.CI_EE_TEST === 'true') {
```

Comment:
I initially thought that I could use the command that I defined in the ee package for updating the endpoint. However, this would then lead to a situation where that command would return the results of the update, wheres the previous set of adjustments in the caller would have its own result. I could do consolidation logic since it is aiming via `environmentId`, but that seemed a bit messy. Let me know if that would still be preferable to this way of updating the data.

Reply:
@rifont let me hear your thoughts on this as well

Reply:
I think this is a suitable approach for the time being 👍